### DISCUSSION TITLE
credit contributions

### DISCUSSION THREAD

**Discussion by PietroLiuzzo** [SChAth/ScAthiop#1861] [Show and tell]
From https://github.com/BetaMasaheft/Manuscripts/pull/1358#issuecomment-996111365

**Comment by PietroLiuzzo** [comment]
Dear @gnizla, referring to the comment link above: you are soooo right! This is a topic I personally care most about (I have also a publication about to appear on this topic). How do we manage to acknowledge the hard work of everyone in a digital research environment? In a "paper" word this is so difficult to surface in full it simply does not happen, but here digital methods have indeed a possibility to set themselves apart as a much better and revolutionary practice. I think there is wide room for improvement and would love to see discussion in general evolve. Let me tell you what is already in place, although very little. 
The model of the data offers the possibility to do this with the `editor` + `@role` and `change` elements, as well as using the `@resp` attribute where it is allowed, for what is limited to encoded and formalized. This allow a rather precise identification of responsibilities. Of course textual formulation are all possible. 
These elements and attributes and additionally the changes which are precisely listed here in GitHub and surfaced in the web application in different ways. 
The draft [package for print](https://github.com/PietroLiuzzo/makepdf/tree/dema) production prepared for the Demarginalizing project also includes a "computation" of acknowledgements and can be easily extended to authors, and other types of contribution. In a way this is required by the copyright of the data  and thus enters a virtuous cycle going straight to challenge the idea of authorship itself. That is to say, when the print process includes materials transcluded from other sources the above elements are taken into consideration to list contributors even without the authors of the product knowing the exact listing. The example only deals with one example of surfacing this, in the acknowledgement section. given that information is there and can be fetched, there is so much more one could do in each of these contexts but especially in the print output to mark also a difference in the ways authorship and scientific contribution is assigned.

**Reply by gnizla** [reply]
Yes, I also had editor + @role in mind as the best possible option.
